<launch>
<!-- This launch file helps in visually positioning the OWI Robot -->

<!-- Load YAML file -->
<rosparam file="$(find owi_control)/config/params.yaml" />

<!-- Webcam -->
<include file = "$(find usb_cam)/launch/usb_cam-test.launch" />

<!-- Aruco marker recognition -->
<include file = "$(find aruco_ros)/launch/quad.launch" />

<!-- End Effector pose -->
<node name = "object_position_tracking_node" pkg="marker_track" type="marker_track_node" output="log" />

<!-- OWI keyboard Control -->
<node name = "owi_key_controller" pkg="owi_control" type="owi_keyboard_node" output="screen"/>

<!-- Position plotter -->
<node name = "owi_position_plotter" pkg="owi_control" type="owi_positioning_node"/>

<!-- Image view -->
 <!-- <node pkg="rqt_image_view" type="rqt_image_view" name="image_view_debug" args="/posn_output"/> -->
<node name="image_draw" pkg="image_view" type="image_view" respawn="false" output="screen">
<remap from="image" to="/posn_output" />
</node>
</launch>